tools/fragpipe/macros.xml

```diff
@@ -0,0 +1,1161 @@
+<macros>
+    <import>msfragger_macros.xml</import>
+    <token name="@TOOL_VERSION@">20.0</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">fragpipe</requirement>
+        </requirements>
+    </xml>
+
+    <!-- TMT-n uses the following n first labels from TMT-18 -->
+    <token name="@TMT_LABELS@">['126', '127N', '127C', '128N', '128C', '129N', '129C', '130N', '130C', '131N', '131C', '132N', '132C', '133N', '133C', '134N', '134C', '135N']</token>
+    <token name="@iTRAQ-4_LABELS@">['114', '115', '116', '117']</token>
+    <token name="@iTRAQ-8_LABELS@">['113', '114', '115', '116', '117', '118', '119', '121']</token>
+
+    <!--
+      User specifies input file format and selects sample files as are included by the scan_inputs macro
+
+      User must also supply their own manifest file, a TSV file of the format:
+      name, experiment, bioreplicate, data type
+    -->
+    <xml name="samples">
+        <conditional name="scan">
```

Comment:
I guess you want to forbid that users mix input with different formats. 
Maybe use a single input and just document that the inputs need to be of the same format. 

Or just support mzml .. people can use converters. 

Reason: with a conditional this becomes hard to use in a workflow.

Reply:
Sure. I just removed the conditional.